ci-operator/step-registry/openshift-qe/installer/azure/update-allocated-outbound-ports/openshift-qe-installer-azure-update-allocated-outbound-ports-commands.sh

```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
+set -o nounset
+set -o errexit
+set -o pipefail
+
+function run_command() {
+    local CMD="$1"
+    echo "Running Command: ${CMD}"
+    eval "${CMD}"
+}
+
+function cli_Login() {
+    # set the parameters we'll need as env vars
+    AZURE_AUTH_LOCATION="${CLUSTER_PROFILE_DIR}/osServicePrincipal.json"
+    AZURE_AUTH_CLIENT_ID="$(cat ${AZURE_AUTH_LOCATION} | jq -r .clientId)"
+    AZURE_AUTH_CLIENT_SECRET="$(cat ${AZURE_AUTH_LOCATION} | jq -r .clientSecret)"
+    AZURE_AUTH_TENANT_ID="$(cat ${AZURE_AUTH_LOCATION} | jq -r .tenantId)"
+    # az should already be there
+    command -v az
+    az version
+
+    # log in with az
+    if [[ "${CLUSTER_TYPE}" == "azuremag" ]]; then
+        run_command "az cloud set --name AzureUSGovernment" || return 1
+    elif [[ "${CLUSTER_TYPE}" == "azure4" ]]; then
+        run_command "az cloud set --name AzureCloud" || return 1
+    elif [[ "${CLUSTER_TYPE}" == "azurestack" ]]; then
+        AZURESTACK_ENDPOINT=$(cat ${SHARED_DIR}/AZURESTACK_ENDPOINT)
+        SUFFIX_ENDPOINT=$(cat ${SHARED_DIR}/SUFFIX_ENDPOINT)
+        cloud_name=$(< "${CLUSTER_PROFILE_DIR}/cloud_name")
+        if [[ -f "${CLUSTER_PROFILE_DIR}/ca.pem" ]]; then
+            cp "${CLUSTER_PROFILE_DIR}/ca.pem" /tmp/ca.pem
+            cat /usr/lib64/az/lib/python*/site-packages/certifi/cacert.pem >> /tmp/ca.pem
+            export REQUESTS_CA_BUNDLE=/tmp/ca.pem
+        fi
+        az cloud register \
+            -n ${cloud_name} \
+            --endpoint-resource-manager "${AZURESTACK_ENDPOINT}" \
+            --suffix-storage-endpoint "${SUFFIX_ENDPOINT}" 
+        az cloud set -n ${cloud_name}
+        az cloud update --profile 2019-03-01-hybrid
+    else
+        echo "Unexpected cluster: ${CLUSTER_TYPE}"
+        exit 1
+    fi
+
+    echo "$(date -u --rfc-3339=seconds) - Logging in to Azure..."
+    az login --service-principal -u "${AZURE_AUTH_CLIENT_ID}" -p "${AZURE_AUTH_CLIENT_SECRET}" --tenant "${AZURE_AUTH_TENANT_ID}" --output none || return 1
+}
+
+function gatherLBs() {
```

Comment:
Where is this function being used?

Reply:
@vishnuchalla It is not being used in the script right now. I copied it from another script.I think we may need to use it in the later when we have more platform to support like azurestack in that func.

Reply:
@vishnuchalla Let me know this looks good to you and we can go ahead to merge this PR.

Reply:
Lets get it added only when it is actually being used. Not much in favor of keeping dead code.

Reply:
@vishnuchalla agree, take out function gatherLBs.

Reply:
@vishnuchalla  I will not run the job again as I just taking out a not used function. Let's get this merged.